src/geophires_x/Economics.py

```diff
@@ -2148,6 +2148,11 @@ def read_parameters(self, model: Model) -> None:
                                                      " range 0-10. GEOPHIRES will assume default surface plant O&M cost correlation with" +
                                                      " adjustment factor = 1.")
                                 ParameterToModify.value = 1.0
+            if self.HeatStartPrice.value > self.HeatEndPrice.value:
+                s = f'{self.HeatStartPrice.Name} cannot be greater than {self.HeatEndPrice.Name}.  ' \
```

Comment:
Include relevant values in parentheses so user can more easily corroborate with their input params